config.toml

```diff
@@ -0,0 +1,85 @@
+[LOADED_NETWORK]
+loaded_network_directory = "Q:/Model Development/Simwrapper Validation/notebooks/roads/Data"
+
+# The dbf file name is LOAD[TOD].dbf by default
+loaded_network_time = ["AM", "MD", "PM", "EV", "EA"]
+
+# The needed CHAMP output columns
+loaded_network_col_names = ["A", "B", "AT", "FT", "V_1"]
+
+[OBSERVED]

```

Comment:
Since "OBSERVED" is the key, "obs" can be dropped from the variable names nested under it.  I think this config file should eventually be merged with the transit config file, we'll need to do one of the following:
1. create separate keys for each data type (ex. OBSERVED_COUNTS), OR
2. keep a single OBSERVED key but make the variables under it more specific (count_path, count_file, count_data_columns, count_info_columns)